src/analysis/retail/priest/discipline/modules/spells/Expiation.tsx

```diff
@@ -22,12 +23,23 @@ class Expiation extends Analyzer {
   expiationIncrease = 0;
   expiationDamage = 0;
   bonusDamage = 0;
+  testHealing = 0;
+  testExpiationHealing = 0;
+  testMbHealing = 0;
+  testDeathHealing = 0;
```

Comment:
So it looks like we have duplicate variable names and an unused function (onAtonement). Can we remove the duplicate variables and onAtonement?